modules/react/checkbox/lib/CheckBackground.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
 import {createComponent, ErrorType} from '@workday/canvas-kit-react/common';
-import {calc, createStencil, createVars, px2rem, cssVar} from '@workday/canvas-kit-styling';
-import {base, brand, system} from '@workday/canvas-tokens-web';
+import {calc, createStencil, createVars, px2rem} from '@workday/canvas-kit-styling';
+import {brand, system} from '@workday/canvas-tokens-web';
 
 interface CheckBackgroundProps {
   children: React.ReactNode;
   error?: ErrorType;
 }
 
-export const backgroundVars = createVars('inner', 'outer');
+export const checkboxStatusColorVars = createVars('inner', 'outer');
```

Comment:
Why aren't these variables of the stencil? Only stencil vars will get documentation added.